plugins/themes/default/styles/objects/issue_toc.less

```diff
@@ -123,5 +131,14 @@
 		.section {
 			margin: @triple -@triple;
 		}
+
+		.cover {
+				margin-right: @triple;
+				margin-bottom: @triple;
+
+			img {
+				max-height: none;
+			}
+		}
```

Comment:
It looks like there are two-tab indentations rather than one-tab indentations for the properties under `.cover` here and above.